.jenkins/static.groovy

```diff
@@ -13,16 +13,16 @@ def runCI =
 {
     nodeDetails, jobName->
 
-    def prj  = new rocProject('hipBLAS', 'StaticLibrary')
-    prj.paths.build_command = './install.sh -cd --static -p /opt/rocm/lib/cmake'
+    def prj  = new rocProject('hipBLAS', 'static')
+    prj.paths.build_command = './install.sh -cs --compiler=hipcc'
 
     if (env.BRANCH_NAME ==~ /PR-\d+/ && pullRequest.labels.contains("noSolver"))
     {
-        prj.libraryDependencies = ['rocBLAS']
+        prj.libraryDependencies = ['rocBLAS', 'rocPRIM']
```

Comment:
I don't think rocPRIM is needed here, shouldn't be a direct dependency of hipBLAS or rocBLAS, but is for rocSOLVER below. Correct me if I'm wrong.